datalad/downloaders/base.py

```diff
@@ -12,27 +12,26 @@
 
 __docformat__ = 'restructuredtext'
 
-import msgpack
 import os
+import os.path as op
 import sys
 import time
+from abc import (
+    ABCMeta,
+    abstractmethod,
+)
+from logging import getLogger
+from os.path import (
+    exists,
+    isdir,
+)
+from os.path import join as opj
 
-from abc import ABCMeta, abstractmethod
-import os.path as op
-from os.path import exists, join as opj, isdir
+import msgpack
```

Comment:
this change doesn't make sense to me, smells like a bug in isort?

Reply:
apparently isort splits import when things are aliased: https://docs.astral.sh/ruff/faq/#can-i-write-my-own-linter-plugins-for-ruff